### PR TITLE
More code signing docs

### PR DESCRIPTION
**Pull request by MarshallOfSound**
Followup to #12767

### CODE REVIEW

File: docs/tutorial/code-signing.md [hunk 1]
```diff
@@ -1,8 +1,15 @@
 # Code Signing
 
 Code signing is a security technology that you use to certify that an app was 
-created by you. Once an app is signed, the system can detect any change to the 
-app, whether the change is introduced accidentally or by malicious code.
+created by you.
+
+On macOS the system can detect any change to the  app, whether the change is
```
Comment (zeke) [inline docs/tutorial/code-signing.md]: `s/the  app/the app/`

File: docs/tutorial/code-signing.md [hunk 2]
```diff
@@ -1,8 +1,15 @@
...
+introduced accidentally or by malicious code.
+
+On Windows the system assigns a trust level to your code signing certificate which
+if you don't have, or if your trust level is low will cause security dialogs to
```
Comment (zeke) [inline docs/tutorial/code-signing.md]: Suggested rewording:

> On Windows the system assigns a trust level to your code signing certificate. If you haven't yet established a trust level or your trust level is low, security dialogs will appear when users start using your application. Your trust level builds over time, so it's better to start code signing as early as possible.

Comment (zeke) [reply docs/tutorial/code-signing.md]: ☝️ Is there a good page we can link to that explains this in more detail?

Comment (MarshallOfSound) [reply docs/tutorial/code-signing.md]: https://blogs.msdn.microsoft.com/ieinternals/2011/03/22/everything-you-need-to-know-about-authenticode-code-signing/

Looks like Microsoft refer to it as "Reputation" not "trust level"

**Pull request merged**